src/graph/transform/remove_edges.cc

```diff
@@ -0,0 +1,88 @@
+/*!
+ *  Copyright (c) 2019 by Contributors
+ * \file graph/transform/remove_edges.cc
+ * \brief Remove edges.
+ */
+
+#include <dgl/base_heterograph.h>
+#include <dgl/transform.h>
+#include <dgl/array.h>
+#include <dgl/packed_func_ext.h>
+#include <dgl/runtime/registry.h>
+#include <dgl/runtime/container.h>
+#include <vector>
+#include <utility>
+#include <tuple>
+
+namespace dgl {
+
+using namespace dgl::runtime;
+using namespace dgl::aten;
+
+namespace transform {
+
+std::pair<HeteroGraphPtr, std::vector<IdArray>>
+RemoveEdges(const HeteroGraphPtr graph, const std::vector<IdArray> &eids) {
+  std::vector<IdArray> induced_eids;
+  std::vector<HeteroGraphPtr> rel_graphs;
+  const int64_t num_etypes = graph->NumEdgeTypes();
+
+  for (int64_t etype = 0; etype < num_etypes; ++etype) {
```

Comment:
Can we use omp here?

Reply:
I think we would be using inter-batch (or batch-wise) multiprocessing instead of intra-batch (or op-level) multithreading.  @jermainewang found that launching OpenMP does not help much on performance presumably due to overhead of launching OpenMP contexts.

Reply:
I c